MetafileImporter/Logic/vtkSlicerMetafileImporterLogic.cxx

```diff
@@ -187,6 +187,41 @@ void UpdateTransformNodeFromString(vtkMRMLLinearTransformNode* transformNode, co
 }
 
 
+void UpdateStringFromTransformNode(std::string& str, vtkMRMLLinearTransformNode* transformNode)
```

Comment:
Makes sense. Shall I add the analogous method:

```void fromString(vtkMatrix4x4* mat, const std::string& str);```

to https://github.com/Slicer/Slicer/blob/master/Libs/vtkAddon/vtkAddonMathUtilities.h also, replacing the above function?